common/k8s/src/lib.rs

```diff
@@ -76,7 +75,8 @@ pub fn create_k8s_client_default_from_env(
         return Err(K8sError::K8sNotInClusterError());
     }
 
-    let config = Config::from_cluster_env()?;
+    let config = Config::incluster_dns()?;
```

Comment:
why renamed to dns?

Reply:
Because we are using rustls-tls, we have to use the dns instead of from the env: https://docs.rs/kube/latest/kube/struct.Config.html#method.incluster_dns